src/constants/max-file-size-bytes.constant.ts

```diff
@@ -0,0 +1 @@
+export const MAX_FILE_SIZE_BYTES = 12582912;
```

Comment:
Do we want this file named like this? It's only used for dtfs -- would `dtfs-max-file-size-bytes` be better?

Reply:
Agreed, renamed